ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -58,24 +58,17 @@ function TokenList({ onTokenClick }: TokenListProps) {
       },
     ]);
 
-    // TODO: consolidate name & title in token fields to avoid this switch
-    if (tokenSortConfig.sortCallback === 'alphaNumeric') {
-      return sortAssets([...filteredAssets], {
-        key: isEvm ? 'name' : 'title',
-        sortCallback: 'alphaNumeric',
-        order: 'asc',
-      });
-    }
-
     // sort filtered tokens based on the tokenSortConfig in state
     return sortAssets([...filteredAssets], tokenSortConfig);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
     isEvm,
     evmBalances,
     multichainAssets,
     networkFilter,
     currentNetwork.chainId,
     tokenSortConfig,
+    // newTokensImported included in deps, but not in hook's logic
```

Comment:
TY for this comment. Interesting we still need this for the useMemo

Reply:
We might not. I just left it but it as it might be worth it to check if this is actually doing anything